### ISSUE TITLE
Thruk 2.34 Ubuntu 16 Xenial, thruk errors, not loading correctly

### ISSUE THREAD

**Issue by majales** [sni/Thruk#1266]
**Describe the bug**
Cannot load /thruk/ web without errors in console. Seems not loading .css files ( 404 Error ).

**Thruk Version**
Default install from repo. 
libthruk/stable,now 2.34 amd64 [installed,automatic]
thruk/stable,now 2.34-3 amd64 [installed]
thruk-base/stable,now 2.34-3 amd64 [installed,automatic]
thruk-plugin-reporting/stable,now 2.34-3 amd64 [installed,automatic]

**To Reproduce**
Go to 'http:/IP/thruk/'  or curl http:/IP/thruk/
 I've got everytime different result

[root@test-naemon]-[/etc/apache2/conf-available]# curl  http://192.168.10.237/thruk/ -v
*   Trying 192.168.10.237...
* Connected to 192.168.10.237 (192.168.10.237) port 80 (#0)
> GET /thruk/ HTTP/1.1
> Host: 192.168.10.237
> User-Agent: curl/7.47.0
> Accept: */*
> 
< HTTP/1.1 200 OK
< Date: Wed, 29 Mar 2023 14:02:42 GMT
< Server: Apache/2.4.18 (Ubuntu)
< Last-Modified: Fri, 12 Jun 2020 17:34:03 GMT
< ETag: "1c69-5a7e67baff8c0"
< Accept-Ranges: bytes
< Content-Length: 7273
< Content-Type: image/png
< 
�PNG

[root@test-naemon]-[/etc/apache2/conf-available]# curl  http://192.168.10.237/thruk/ -v
*   Trying 192.168.10.237...
* Connected to 192.168.10.237 (192.168.10.237) port 80 (#0)
> GET /thruk/ HTTP/1.1
> Host: 192.168.10.237
> User-Agent: curl/7.47.0
> Accept: */*
> 
< HTTP/1.1 200 OK
< Date: Wed, 29 Mar 2023 14:03:38 GMT
< Server: Apache/2.4.18 (Ubuntu)
< Last-Modified: Fri, 12 Jun 2020 11:13:08 GMT
< ETag: "47e-5a7e1296c5900"
< Accept-Ranges: bytes
< Content-Length: 1150
< Content-Type: image/vnd.microsoft.icon
< 
 h

[root@test-naemon]-[/etc/apache2/conf-available]# curl  http://192.168.10.237/thruk/ -v
*   Trying 192.168.10.237...
* Connected to 192.168.10.237 (192.168.10.237) port 80 (#0)
> GET /thruk/ HTTP/1.1
> Host: 192.168.10.237
> User-Agent: curl/7.47.0
> Accept: */*
> 
< HTTP/1.1 302 Found
< Date: Wed, 29 Mar 2023 14:03:44 GMT
< Server: Apache/2.4.18 (Ubuntu)
< Location: http://192.168.10.237/thruk/cgi-bin/login.cgi?thruk/
< Content-Length: 316
< Content-Type: text/html; charset=iso-8859-1
< 
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>302 Found</title>

**Expected behavior**
Thruk web should load without errors

**Screenshots**

![Snímek obrazovky pořízený 2023-03-29 16-32-38](https://user-images.githubusercontent.com/11310917/228573209-c469f2a0-e2b0-450c-9852-1a39b6d6af87.png)
![Snímek obrazovky pořízený 2023-03-29 16-37-09](https://user-images.githubusercontent.com/11310917/228573930-c41ae0ec-58c6-4306-88bb-68430c01997b.png)

**Desktop (please complete the following information):**
 - Browser chromium 111.0.5563.64, Firefox 111.0.1

**Comment by majales** [comment]
I know it is pretty old release.. it is xenial and packages are form here https://labs.consol.de/repo/stable/ubuntu/dists/xenial/main/binary-amd64/

**Comment by majales** [comment]
Finally I've got Thruk webinterface working correctly. It was my mistake, but this time it was a bit cryptic one. Apache startup script on Ubuntu 16 LTS is old one, placed in /etc/init.d/  . For some strange reason it reads content of /root/.profile and my changes in root profile ( I did changes back in 2018 ) caused everything going smoothly up until now when I tried install Thruk on apache with mod_fcgid. When I replace /root/.profile with original file, it just started working. We can close this issue.
